### DISCUSSION TITLE
First Issue friendly?

### DISCUSSION THREAD

**Discussion by kiridarivaki** [vanjs-org/van#279] [General]
Hello everyone! I am a computer science student looking for an OSS to work on, for a course assignment i have. I am currently learning javascript, so i haven't gotten familiar with any frameworks yet and i would like to try contributing to this project! Do you think it will be possible for me? Do you have any ideas for good first issues? It is not required to get my pull request accepted for this course, but i want to try my best:)

**Comment by Tao-VanJS** [comment]
Hi @kiridarivaki,

Thank you so much for your interest in VanJS!

Contributions are welcome and will be greatly appreciated! In terms of which areas to contribute, for the core library, since the library has simplicity as its prominent principle, the bar will be high as it's quite challenging to add new things while maximally maintaining the simplicity. On the other hand, VanJS community has contributed substantially to [VanJS add-ons](https://vanjs.org/#community-add-ons) - 3rd-party libraries on top of VanJS that enhance and extend its functionality. Meanwhile, it will also be greatly appreciated just to build real world applications on top of VanJS, we have a curated list in VanJS website for [these apps](https://vanjs.org/demo#community-examples). These are also very important as user feedback is very valuable to us, lots of feature requests and bugs are coming from users building apps on top of VanJS.

Thanks again and hope VanJS can provide a good developer experience to you!
